tests/integration/actions/doc/test_welcome_interactive_noee.py

```diff
@@ -6,7 +6,6 @@
 
 from .base import BaseClass
 
-
```

Comment:
There's an option to make this post-import two-line separator consistent regardless of the var assignment expressions being present. I'd prefer for us to use that.

Reply:
nice, adding.